bindings/java/src/main/java/org/ucx/jucx/ucs/UcsConstants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+package org.ucx.jucx.ucs;
+
+public class UcsConstants {
+
+    /**
+     * Thread sharing mode
+     *
+     * <p>Specifies thread sharing mode of an object.
+     */
+    public enum UcsThreadMode {
+        UCS_THREAD_MODE_SINGLE,     /**< Only the master thread can access
+                                         (i.e. the thread that initialized the context;
+                                         multiple threads may exist and never access) */
+        UCS_THREAD_MODE_SERIALIZED, /**< Multiple threads can access, but only one at a time */
```

Comment:
This mode copies MPI API, I think we can add for Java only single and multi

Reply:
This need to have the same order as in UCX - thus have the same value for modes after this.

Reply:
java doesn't support enum with specific values?

Reply:
seems like it's possible: https://stackoverflow.com/questions/8811815/is-it-possible-to-assign-numeric-value-to-an-enum-in-java

Reply:
Totally possible. But we decided to set all the constants from JNI. Ok i'll rewrite it to set from JNI.